tsdb/docs/usage.md

```diff
@@ -1,6 +1,6 @@
 # Usage
 
-TSDB can be - and is - used by other applications such as [Cortex](https://cortexmetrics.io/) and [Thanos](https://thanos.io/).
+TSDB can be - and is - used by other applications such as [Cortex](https://cortexmetrics.io/), [Thanos](https://thanos.io/) and [Grafana Mimir](https://grafana.com/oss/mimir/).
```

Comment:
```suggestion
TSDB can be - and is - used by other applications such as [Cortex](https://cortexmetrics.io/), [Thanos](https://thanos.io/), and [Grafana Mimir](https://grafana.com/oss/mimir/).
```

(My impression is that we prefer the Oxford comma.)

Reply:
By we you mean the Prometheus project? I wasn't sure.